### ISSUE TITLE
Use of Bounding Box in the code

### ISSUE THREAD

**Issue by Nabil-Miri** [PRBonn/LocNDF#9]
Hello,

I would like to know what does the BBox for the data?
Does it limit the scan points used around the key pose (or scan idx selected in the config file)

Moreover, in the generated hparams.yaml, it has also BBox info. What does it mean?

Thanks.

**Comment by louis-wiesmann** [comment]
Yes i only used the points which lie in the BBox to train the model. It is at the same time used to normalize the input coordinates.

**Comment by Nabil-Miri** [comment]
Hi Louis,
Thanks for your reply.

Is it possible to use 3D data from Azure camera using the same model ?

Sample of the data comparing both: [Azure data in the middle]

![Screenshot from 2024-02-22 11-52-46](https://github.com/PRBonn/LocNDF/assets/75530842/8dbd6219-30e0-49cd-812a-b04a3f2fd7a3)

Using the same dataloader gave weird mesh results. What would be the main changes needed to adapt my data to the model ?

**Comment by louis-wiesmann** [comment]
Hard to grasp in this image.

Theoretically I don't see any problem. You should adapt the bounding box and normalization to the scale of your scene. But i would not recommend going with to big scenes. Then tuning the learning rate and the loss parameter. 
The only problem i could see is when the sensor is having way more noise. I'm not completely sure how the network is reacting there. Guess one needs to play with the parameters to make sure it is averaging the noise out.

**Comment by Nabil-Miri** [comment]
For my case the depth points are collected from the Azure in a certain pose, then it is rotated a bit and the goal is to create a NDF model for the room or environment. So do I need to use the bounding box as my environment is relatively small?

Can you please point out in the code where normalization is done to change it.

**Comment by louis-wiesmann** [comment]
You just have to change the bounding box size in the config. The normalization is based on the bounding box [check here](https://github.com/PRBonn/LocNDF/blob/main/src/loc_ndf/models/models.py#L28). The bounding box should reflect the scene you want to observe. Taking 50m for an indoor scene might be too big and you would lose finer details in the map.

**Comment by Nabil-Miri** [comment]
Thank you Louis for the information. I will try your suggestions.
